features/facebookImport/test/ministory/messages-analysis.test.js

```diff
@@ -49,7 +49,7 @@ describe("Messages analysis for export with messages", () => {
         );
         ({ status } = analysisResult);
         analysisStory = new MessagesMinistory(facebookAccount);
-        analysisData = getAnalysisData(facebookAccount);
```

Comment:
But you see, this is the problem. In a sea of 60 files changed, it's easy to miss a simple (relatively) variable renaming, either by you or the reviewer. That's why I advocate for simpler, more atomic PRs.

Reply:
wise words indeed 🙏